include/fmt/core.h

```diff
@@ -392,11 +392,12 @@ template <typename Char> class basic_string_view {
     the size with ``std::char_traits<Char>::length``.
     \endrst
    */
-#if __cplusplus >= 201703L  // C++17's char_traits::length() is constexpr.
-  FMT_CONSTEXPR
+// C++17's char_traits::length() is constexpr.
```

Comment:
Looks good but please move this comment back to the next line to avoid interfering with an apidoc comment above and apply clang-format.